trieste/acquisition/rule.py

```diff
@@ -290,7 +293,75 @@ def acquire(
             )
 
         acquisition_function = self._builder.prepare_acquisition_function(datasets, models)
+
         point = _optimizer.optimize(acquisition_space, acquisition_function)
         state_ = TrustRegion.State(acquisition_space, eps, y_min, is_global)
 
         return point, state_
+
+
+class BatchAcquisitionRule(AcquisitionRule[None, SearchSpace]):
+    """ Implements a acquisition rule for a batch of query points """
+
+    def __init__(self, num_query_points: int,
```

Comment:
you mentioned in (the other PR)[https://github.com/secondmind-labs/trieste/pull/53/files#r522213555] that you think the batch size should be passed to the rule. Can you elaborate on why?

Reply:
it seems more intuitive to me. You set the rule with the batch size: the rule is the evaluate the B set of points that optimize the acq function. 
The alternative, where the acq fct defines the batch size, seems backwards. You would then use the properties of the acqfct to define the optimizer for instance.